src/javascript/app_2/App/Components/Elements/Calendar/panels/types.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+export const CalendarPanelTypes = {
+    calendarDate    : PropTypes.string, 
```

Comment:
@kellybinary 
plz change those variable names which are not a function to `snake_case`